formats/placeholder/frontend/schema.json

```diff
@@ -0,0 +1,147 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "base_path",
+    "format",
+    "locale"
+  ],
+  "properties": {
+    "base_path": {
+      "type": "string"
+    },
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "format": {
+      "type": "string",
+      "enum": [
+        "placeholder"
```

Comment:
Unfortunately, that would require a code change as the `format` enum is [automatically added based upon the format](https://github.com/alphagov/govuk-content-schemas/blob/master/lib/govuk_content_schemas/schema_combiner.rb#L23). Given that you're creating a precedent for variable values for `format`, would you be happy to try it in a separate PR?